src/test/testng.xml

```diff
@@ -6,6 +6,10 @@
         <groups>
             <define name="include-group">
                 <include name="active" />
+                <include name="include-test-one" />
+                <include name="include-test-two" />
```

Comment:
where are these to defined? Are these supposed to be the pattern tests? or why are these here?

Reply:
that was from the original example - is removed